tests/lib/rules/component-name-in-template-casing.js

```diff
@@ -854,6 +872,39 @@ tester.run('component-name-in-template-casing', rule, {
         </script>
       `,
       errors: ['Component name "TheComponent" is not kebab-case.']
+    },
+    {
+      code: `
+        <template>
+          <router-view />
+        </template>
+      `,
+      options: ['PascalCase', { globals: ['RouterView'] }],
+      output:
+        '\n        <template>\n          <RouterView />\n        </template>\n      ',
```

Comment:
Please use `` `template literals` `` with actual newlines instead of normal `'strings'` with escaped newlines.

Reply:
Done.